cpp/src/IceGrid/Internal.ice

```diff
@@ -389,7 +389,7 @@ module IceGrid
     interface InternalRegistry extends FileReader
     {
         /// Register a node with the registry. If a node with the same name is already registered,
-        /// <code>registerNode</code> overrides the existing registration only when the previously
+        /// registerNode overrides the existing registration only when the previously
```

Comment:
I would prefer "this operation" when referencing the operation we're describing. Same for registerReplica below.